articles/storage/storage-python-how-to-use-table-storage.md

```diff
@@ -102,13 +102,12 @@ use the **TableBatch** class. When you do want to submit the
 batch, you call **commit\_batch**. Note that all entities must be in the same partition in order to be changed as a batch. The example below adds two entities together in a batch.
 
 ```python
-from azure.storage.table import TableBatch
-batch = TableBatch()
+batch = table_service.begin_batch()
```

Comment:
@asears Hi Andrew, I'll be requesting that this PR be closed in one week if no further action is taken.